src/builder/util.js

```diff
@@ -791,6 +791,20 @@ function getStringWidth(text) {
   return stringWidth(text.replace(emojiRegex, "  "));
 }
 
+function hasIgnoreComment(path) {
+  const node = path.getValue();
+  return hasNodeIgnoreComment(node);
+}
+
+function hasNodeIgnoreComment(node) {
+  return (
+    node &&
+    node.comments &&
+    node.comments.length > 0 &&
+    node.comments.some(comment => comment.value.trim() === "prettier-ignore")
```

Comment:
FYI `[].some(() => true) === false`, so `node.comments.length > 0` is not needed other than for explicitness.

Reply:
True. I just moved this code though :wink: